Packs/CSCDomainManager/Integrations/CSCDomainManager/CSCDomainManager.py

```diff
@@ -0,0 +1,553 @@
+        outputs_prefix='CSCDomainManager.Domain',
+        outputs_key_field='QualifiedDomainName',
+        outputs=domains_list
+    )
+    return results
+
+
+def csc_domains_availability_check_command(client: Client, args) -> CommandResults:
+    """
+    Returning a list of available domains with the applied filters
+
+    Args:
+        client: CSCDomainManager client
+        args: demisto.args()
+
+    Returns:
+        A list of available domains with the applied filters
+    """
+    domain_names = args.get('domain_name')
+    params = f'qualifiedDomainNames={domain_names}'
+    available_domains_results = (client.send_get_request("/availability", params)).get('results')
+
+    hr_output = get_domains_availability_check_hr_fields(available_domains_results)
+
+    results = CommandResults(
+        readable_output=tableToMarkdown('Domains Availability', hr_output, headers=HR_HEADERS_FOR_AVAILABILITY),
+        outputs_prefix='CSCDomainManager.Domain.Availability',
+        outputs=available_domains_results
+    )
+    return results
+
+
+def csc_domains_configuration_list_command(client: Client, args) -> CommandResults:
+    """
+    Returning a list of domains configurations with the applied filters
+
+    Args:
+        client: CSCDomainManager client
+        args: demisto.args()
+
+    Returns:
+        A list of domains configurations with the applied filters
+    """
+    args_copy = copy.deepcopy(args)
+    args_copy['page'] = args_copy.get('page') or DEFAULT_PAGE_SIZE_CONFI
+    args_copy['page_size'] = args_copy.get('page_size') or DEFAULT_PAGE_SIZE_SEARCH
+
+    if args_copy.get('limit'):
+        args_copy['page'] = '1'
+        args_copy['page_size'] = args_copy.get('limit')
+
+    params_results = create_params_string(args_copy)
+    configurations_results = client.send_get_request(url_suffix="/domains/configuration", params=params_results)
+
+    configurations_list = configurations_results.get('configurations', [])
+    configurations_with_required_fields = get_domains_configurations_hr_fields(configurations_list)
+
+    results = CommandResults(
+        readable_output=tableToMarkdown('Filtered Configurations',
+                                        configurations_with_required_fields,
+                                        headers=HR_HEADERS_FOR_DOMAIN_CONFI_LIST),
+        outputs_prefix='CSCDomainManager.Domain.Configuration',
+        outputs_key_field='CSCDomainManager.Domain.Configuration.Domain',
+        outputs=configurations_list
+    )
+    return results
+
+
+def domain(client: Client, args, reliability) -> CommandResults:
+    """
+    Gets the domain
+
+    Args:
+        client: CSCDomainManager client
+        args: demisto.args()
+        reliability: The source reliability. Default set to A.
+
+    Returns:
+       the domain information
+    """
+    qualified_domain_name = args.get('domain')
+    domain_json = client.send_get_request(url_suffix=f"/domains/{qualified_domain_name}", params="")
```

Comment:
instead of calling .send_get_request from outside the client class, lets make it an internal method to client, each url_suffix should get its own function. this should be `client.get_qualified_domamin_name(qdn)`